### ISSUE TITLE
Please add a @NgModule annotation.

### ISSUE THREAD

**Issue by AlexanderKozhevin** [swimlane/ngx-dnd#89]
An error during build process.

```
ERROR in : Unexpected value 'NgxDnDModule in /Users/alexanderkozhevin/Desktop/virtus/portalFrontend/node_modules/@swimlane/ngx-dnd/release/index.js' imported by the module 'AppModule in /Users/alexanderkozhevin/Desktop/virtus/portalFrontend/src/app/app.module.ts'. Please add a @NgModule annotation.
```

```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Sorry, we will not be able to answer every support request.  Please consider other venues for support requests
```

**Current behavior**
I'm using angular-cli. Ng-serve works fine. When when I try to make final build *npm run build*. It throws an error.

* **ngx-dnd version:** 4.0.2

* **Angular version:** 5.2.0

**Comment by AlexanderKozhevin** [comment]
With version *4.0.0* everything works fine.

**Comment by ThisIsnonVoid** [comment]
Thank you @AlexanderKozhevin - works just fine with 4.0.0. Only had to add the line 
`(window as any).global = window;` in the polyfills.ts for production build.

**Comment by realmar** [comment]
Same problem here with all 4.x.x versions (4.0.0, 4.0.1, 4.0.2)

**Comment by jeanpham** [comment]
Hi, I have the same problem. Anybody a solution? Thanks!

**Comment by duard** [comment]
Just change it to ngx-dnd version: 4.0.0

**Comment by jeanpham** [comment]
@duard Thanks! That worked! What is exactly the problem with 4.0.2?

**Comment by ssuperczynski** [comment]
Since version 4.0.1 to 4.0.2 the only change is mine #84, but is not related with this one.
What I noticed is that ngx-dnd build is not AOT compatible, and there are more errors need to be fixed to work under Angular 6.

**Comment by Hypercubed** [comment]
Yes...  please everyone be patient.  We are working on a version built with angular cli from the beginning.  Should be done soon.

**Comment by fklingler** [comment]
There also seem to have a problem with the typings at the moment.

```
client/app/admin/admin.module.ts(6,30): error TS7016: Could not find a declaration file for module '@swimlane/ngx-dnd'. '<project path>/node_modules/@swimlane/ngx-dnd/release/index.js' implicitly has an 'any' type.
  Try `npm install @types/@swimlane/ngx-dnd` if it exists or add a new declaration (.d.ts) file containing `declare module '@swimlane/ngx-dnd';`
```

The `package.json` file indicates this: `"typings": "release/index.d.ts"`, but there is no `release/index.d.ts` file.

Thanks for giving us updates :)

**Comment by Hypercubed** [comment]
Can you please give the next version a try: `npm install @swimlane/dragula @swimlane/ngx-dnd@next`

**Comment by litchian18** [comment]
I have same issue on  version  4.0.2

**Comment by Hypercubed** [comment]
Just published 5.0.2, if you still see this issue please reopen.

**Comment by issamalshamaly** [comment]
Hi, Yes when I tried to build the project I got an error with ngx-dnd library
So I updated the library to the last version I think 5.0.6 but unfortunately I got 
An error of can't find dragula module in some component by seeing the 
Old version I found a reference is missing l wrote it I got the final error
And it was global undefined l solved  it and it works fine in Ng serve and Ng build.
Angular version is 6.
I suggest to update swimlan and if error appears I will try to help you.

I will follow this issue.

Thank you.
